docs/concepts/rules.md

```diff
@@ -86,6 +85,16 @@ as a single developer account can and frequently will request/rotate between
 multiple different signing certificates and entitlements. This is an even more
 powerful rule with broader reach than individual certificate rules.
 
+### Compiler/Transitive Rules
+
+The transitive whitelist capability of Santa can automatically whitelist any files that are created by a set of specified binaries.  A typical use-case is allowing any binaries compiled with XCode on developer machines to execute, as it would be slow and impractical to use other rule types to permit these.
+
+To begin using transitive whitelisting, `EnableTransitiveWhitelisting` should be set to true and Compiler rules (rules with the policy `ALLOWLIST_COMPILER`) should be added to indicate the binaries which will be writing the new files to be whitelisted.  Santa will create and manage Transitive rules in its database automatically, they cannot be created directly.
```

Comment:
Minor tweaks to use the new preferred naming and the non-deprecated configuration key:

```suggestion
To begin using transitive allowlisting, `EnableTransitiveRules` should be set to true and Compiler rules (rules with the policy `ALLOWLIST_COMPILER`) should be added to indicate the binaries which will be writing the new files to be allowlisted.  Santa will create and manage Transitive rules in its database automatically, they cannot be created directly.
```